README.md

```diff
@@ -1,13 +1,74 @@
-# Taskmaster [![42](https://i.imgur.com/9NXfcit.jpg)](i.imgur.com/9NXfcit.jpg)
 
-Taskmaster is a 42 project. It is an implementation of a **process supervisor** and aims at getting familiar with job control mechanisms on UNIX-like operating systems.
+<br />
+<p align="center">
+  <a href="https://github.com/pscott/TaskMaster">
+    <img src="assets/logo.png" alt="Logo" width="90" height="90">
+  </a>
 
----
+  <p align="center">
+    <img src="assets/title.png" alt="TaskMaster" width="280" height="70">
+  </p>
+
+  <p align="center">
+    A process control system for UNIX.
+    <br />
+    <a href="https://github.com/pscott/TaskMaster">View Demo</a>
+    ·
+    <a href="https://github.com/pscott/TaskMaster/issues">Report Bug</a>
+    ·
+    <a href="mailto:abarthel@student.42.fr">Request Feature</a>
+  </p>
+</p>
+
+[![Contributors][contributors-shield]][contributors-url]
+
+
+
+## Table of Contents
+
+* [About the Project](#about-the-project)
+  * [Built With](#built-with)
+* [Getting Started](#getting-started)
+  * [Prerequisites](#prerequisites)
+  * [Installation](#installation)
+* [Usage](#usage)
+* [Roadmap](#roadmap)
+* [Contributing](#contributing)
+* [Contact](#contact)
+* [Acknowledgements](#acknowledgements)
+
+
+
+## About The Project
+
+Taskmaster is a **[42 project]** client/server implementation that allows users to monitor and control their processes. It aims at getting familiar with job control mechanisms on UNIX-like operating systems.
```

Comment:
When I saw square brackets my brain went "oh, missing link" haha. I'd go for either removing square brackets, or adding a link to 42.fr :)

Reply:
The intend was to say this in another way:
` Taskmaster - a 42 project - is a client/server implementation that allows use[...]`

Reply:
Yup using dashes makes it clearer IMO :)